DESCRIPTION

```diff
@@ -47,14 +47,14 @@ Suggests:
     scda.2022 (>= 0.1.3),
     sparkline,
     stringr,
-    teal (>= 0.12.0),
+    teal (>= 0.12.0.9012),
```

Comment:
current @main version is `.9022`. Are we suppose to keep the current version or last "breaking" change? 

We are going to merge https://github.com/insightsengineering/teal/pull/806 today so I suggest to even upgrade to `.9023`